### PR TITLE
fixed a potential bug

### PR DESCRIPTION
**Pull request by wiseaidev**
Related to #322

The reason for removing `six`, assuming you are using python 3.7+, is that it has the same memory footprint as the built-in range method:

```python
>>> import six
>>> import sys
>>> from six.moves import xrange
>>> x = xrange(1,10000)
>>> a = range(1,10000)
>>> print (sys.getsizeof(a))
48
>>> print (sys.getsizeof(x))
48
```

Therefore, no benefits from using it instead of `range`.

### CODE REVIEW

File: aredis_om/model/model.py [hunk 1]
```diff
@@ -1179,50 +1179,32 @@ def find(cls, *expressions: Union[Any, Expression]) -> FindQuery:
...
-                        map(to_string, res[i + fields_offset][::2]),
-                        map(to_string, res[i + fields_offset][1::2]),
+                    zip(
+                        map(to_string, res[i + offset][::2]),
+                        map(to_string, res[i + offset][1::2]),
                     )
-                )
             )
-
-            try:
```
Comment (dvora-h) [inline aredis_om/model/model.py:1210]: Nice catch!

**Comment by dvora-h** [comment]
@wiseaidev Great PR, thanks!

Can you solve the linter issue so that all the tests will run and we can merge it?

**Comment by wiseaidev** [comment]
Hey @dvora-h, done!

**Comment by wiseaidev** [comment]
I see what's wrong, working on it!

**Review by dvora-h — APPROVED** [review]

**Comment by dvora-h** [comment]
@wiseaidev Thanks for the quick fixes!

**Pull request merged**